Exiled.Permissions/Extensions/Permissions.cs

```diff
@@ -88,6 +88,14 @@ public static void Create()
         /// </summary>
         public static void Reload()
         {
+            if (ServerStatic.PermissionsHandler == null)
+            {
+                Log.Error("Your Remote Admin config is broken. You have to fix it because the game won't even start with a broken config.");
+
+                // Then do nothing.
```

Comment:
Do we need this comment? Ahah

Reply:
I mean, it says what it is lmao

Reply:
![](https://cdn.discordapp.com/emojis/799024321049657344.png)

Reply:
Should we add `throw new YouAreDumpException("You borked your Remote Admin configs, you monkey!");`?
![Troll](https://user-images.githubusercontent.com/5269236/108887231-40921c80-760a-11eb-9a4b-3a7dd1bd5e9b.gif)

Reply:
I think everything should be fine now.